libs/ngxtension/call-apply/src/call-apply.ts

```diff
@@ -0,0 +1,44 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+const error_this = function () {
+	throw new Error(
+		`DON'T USE this INSIDE A FUNCTION CALLED BY | call OR | apply IT MUST BE A PURE FUNCTION!`
+	);
+};
+const NOTHIS = !('Proxy' in window)
+	? Object.seal({})
+	: new Proxy(
+			{},
+			{
+				get: error_this,
+				set: error_this,
+				deleteProperty: error_this,
+				has: error_this,
+			}
+	  );
+
+@Pipe({
+	name: 'call',
+	pure: true,
+	standalone: true,
+})
+export class CallPipe implements PipeTransform {
+	transform(value: any, args?: Function): any {
```

Comment:
```suggestion
	transform<TFunction extends (...args: any[]) => any>(value: Parameters<TFunction>[0], fn: TFunction): ReturnType<TFunction> {
```

We can use the same approach for `call`

Reply:
there is no way to pass more than 1 param to call, because you do `{{ par | call: fn }}` I've just updated my PR with a commit containing your suggestion, let's me know if you need any other update...